gateway/handlers/certificates_handler.go

```diff
@@ -0,0 +1,89 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"path"
+
+	"github.com/gorilla/mux"
+)
+
+var keyMap = map[string]string{
+	"callback": "http-signing-public-key",
+}
+
+type KeyType struct {
+	Id  string `json:"id"`
+	PEM string `json:"pem"`
+}
+
+type SecretsReader interface {
+	Read(key string) (string, error)
+}
+
+type FileSecretsReader struct {
+	SecretMountPath string
+}
+
+func (f *FileSecretsReader) Read(key string) (string, error) {
```

Comment:
Reminds me of https://github.com/openfaas/openfaas-cloud/blob/master/sdk/secrets.go#L13

Reply:
I think there are some other similar bits of code in the project for reading secrets, can you have a quick look?